From 3509c5c0c4d7256e9ce70557cb44a38c1c644e6a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?G=C3=BCnther=20Deschner?= <gd@samba.org>
Date: Thu, 5 Apr 2012 14:05:00 +0200
Subject: [PATCH 234/321] nsswitch: disable HAVE_PAM_RADIO_TYPE handling until
 proper PAM_RADIO_TYPE handling is available.

 This is needed that gdm doesn't crash.

Guenther
---
 nsswitch/pam_winbind.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/nsswitch/pam_winbind.c b/nsswitch/pam_winbind.c
index 8f5ad507523..0a5098d01ff 100644
--- a/nsswitch/pam_winbind.c
+++ b/nsswitch/pam_winbind.c
@@ -807,7 +807,9 @@ static int wbc_auth_error_to_pam_error(struct pwb_context *ctx,
 	return pam_winbind_request_log(ctx, ret, username, fn);
 }
 
-#if defined(HAVE_PAM_RADIO_TYPE)
+#if 0
+/* #if defined(HAVE_PAM_RADIO_TYPE) currently disabled until proper
+ * PAM_RADIO_TYPE is implemented - gd */
 static bool _pam_winbind_change_pwd(struct pwb_context *ctx)
 {
 	struct pam_message msg, *pmsg;
-- 
2.14.3

